mapper/mapper.go

```diff
@@ -276,9 +276,13 @@ func CreateVersionsList(ctx context.Context, d dataset.Model, edition dataset.Ed
 
 		var correctionReasons []string
 		const correctionAlertType = "correction"
-		for _, alert := range *ver.Alerts {
-			if alert.Type == correctionAlertType {
-				correctionReasons = append(correctionReasons, alert.Description)
+		if *ver.Alerts != nil {
+			for _, alert := range *ver.Alerts {
+				if &alert != nil {
+					if alert.Type == correctionAlertType {
+						correctionReasons = append(correctionReasons, alert.Description)
+					}
+				}
 			}
 		}
 		version.Reasons = correctionReasons
```

Comment:
would you want to assign an empty slice to Reasons if there were no alerts? or could we move it inside the `if *ver.Alerts != nil` block?

Reply:
Moved :)